Library/Homebrew/tap.rb

```diff
@@ -525,8 +526,6 @@ def to_hash
 
   # Hash with tap formula renames
   def formula_renames
-    require "json"
-
     @formula_renames ||= if (rename_file = path/"formula_renames.json").file?
```

Comment:
This could perhaps use a constant which is included in the `HOMEBREW_TAP_JSON_FILES` constant?

Reply:
Good idea. I've got to rerun CI after the core PR is merged anyway so I may as well make these changes as well.